### PR TITLE
Jicxer dynamo cmdline

### PR DESCRIPTION
**Pull request by Jicxer**

### CODE REVIEW

**Review by endepointe — CHANGES_REQUESTED** [review]
merge branch dynamo-with-iam and resubit pr. Need to clean up the merge conflicts to get a stable branch

File: default.config
Comment (endepointe) [inline default.config:1]: remove this file

File: 0057fb91-77e6-4a14-90b1-bfa266d38e00
Comment (endepointe) [inline 0057fb91-77e6-4a14-90b1-bfa266d38e00:1]: remove this file

File: 277b2d03-ddd5-494a-a048-0810582a1fe7
Comment (endepointe) [inline 277b2d03-ddd5-494a-a048-0810582a1fe7:1]: remove this

File: d9264065-d8d4-44f7-bae1-ee09e909e016
Comment (endepointe) [inline d9264065-d8d4-44f7-bae1-ee09e909e016:1]: remove

File: src/config.rs
Comment (endepointe) [inline src/config.rs:1]: merge the dynamo-with-iam src/config.rs file

File: src/firehosesdk.rs
Comment (endepointe) [inline src/firehosesdk.rs:1]: delete this

File: src/util.rs
Comment (endepointe) [inline src/util.rs:1]: move this function:
                pub async fn send_dynamodb(config: Config) 

into srd/dynamosdk.rs

File: src/producer.rs
Comment (endepointe) [inline src/producer.rs:1]: merge the dynamo-with-iam branch to overwrite this file

**Review by endepointe — CHANGES_REQUESTED** [review]
perform the changes listed and hopefully the conflicts become manageable

**Comment by Jicxer** [comment]
Closed pull request and opened a new one with the requested changes. Merging into dynamo-with-iam branch before dev.

**Pull request closed without merging**